src/operator/src/metrics.rs

```diff
@@ -93,4 +93,11 @@ lazy_static! {
         ]
     )
     .unwrap();
+    pub static ref BULK_REQUEST_ROWS: HistogramVec = register_histogram_vec!(
+        "greptime_table_operator_bulk_insert_message_rows",
+        "table operator bulk inserts message rows",
+        &["type"],
+        linear_buckets(0.0, 10000.0, 10).unwrap()
+    )
+    .unwrap();
```

Comment:
The buckets num is too large. better to use exponential buckets.

Reply:
fixed in 065cf78f1cea9ae9f502084c4c498ddadb3d3a94